sdks/java/core/src/main/java/org/apache/beam/sdk/PipelineRunner.java

```diff
@@ -25,15 +25,14 @@
 import org.apache.beam.sdk.util.InstanceBuilder;
 
 /**
- * A {@link PipelineRunner} can execute, translate, or otherwise process a
- * {@link Pipeline}.
+ * A {@link PipelineRunner} runs a {@link Pipeline}.
  *
- * @param <ResultT> the type of the result of {@link #run}.
+ * @param <ResultT> the type of the result of {@link #run}, often a handle to a running job.
  */
 public abstract class PipelineRunner<ResultT extends PipelineResult> {
 
   /**
-   * Constructs a runner from the provided options.
+   * Constructs a runner from the provided {@link PipelineOptions}.
```

Comment:
Perhaps it is worth adding an additional note that subclasses of `PipelineRunner` must declare this method as well?